test/api/peer.transactions.main.js

```diff
@@ -77,6 +77,18 @@ describe('POST /peer/transactions', function () {
 			});
 	});
 
+	it('using data field should be ok', function (done) {
+		var account = node.randomAccount();
+		var transaction = node.lisk.transaction.createTransaction(account.address, 1, node.gAccount.password);
```

Comment:
Ok fair point. We will work on adding support for this in `lisk-js`: https://github.com/LiskHQ/lisk-js/issues/52